apps/src/client.rs

```diff
@@ -89,7 +89,18 @@ pub fn connect(
     // Create the configuration for the QUIC connection.
     let mut config = quiche::Config::new(args.version).unwrap();
 
-    config.verify_peer(!args.no_verify);
+    if let Some(ref trust_origin_ca_pem) = args.trust_origin_ca_pem {
+        config
+            .load_verify_locations_from_file(trust_origin_ca_pem)
+            .map_err(|e| {
+                ClientError::Other(format!(
+                    "error loading origin CA file : {}",
+                    e
+                ))
+            })?;
+    } else {
+        config.verify_peer(!args.no_verify);
```

Comment:
Why put this in the `else` block? I could see `--no-verify` being used together with the new option for testing. Say for example I connect to a server with the new option enabled, but it still doesn't work, I might try to append `--no-verify` to quickly check if that works, but with the current implementation that would also require removing the other option which can be annoying.